airflow/www/views.py

```diff
@@ -3271,6 +3271,58 @@ def list(self):
         )
 
 
+class ProvidersView(AirflowBaseView):
```

Comment:
```suggestion
class ProviderView(AirflowBaseView):
```

I think this should be singular.